### ISSUE TITLE
Formating issue with location levels

### ISSUE THREAD

**Issue by David-LRL** [USACE/cwms-data-api#271]
Previous release returned a json object by default.  If you specify a format you get an error if you don't specify a format you get the results below.

Sample Query:
https://wm.lrl.ds.usace.army.mil:8243/lrl-data/levels?name=CaesarCreek.Elev.Inst.0.Bottom%20of%20Flood%20Control&unitSystem=EN&office=LRL&datum=NGVD29&begin=2022-11-04T06:00:00Z&end=2022-11-04T06:00:00Z&timezone=America/Louisville

Output:
#Processed At	coe-lrluwd04lrl:H2CWMSP1
#Time Of Query	07-Nov-2022 18:55 UTC
#Process Query	547 milliseconds
#Format Output	0 milliseconds
#Requested Start Time	04-Nov-2022 06:00 America/Louisville
#Requested End Time	04-Nov-2022 06:00 America/Louisville
#Requested Format	TAB
#Requested Office	LRL
#Requested Names	CaesarCreek.Elev.Inst.0.Bottom of Flood Control
#Requested Units	EN
#Requested Datums	NGVD29
#Total Location Levels Cataloged	1
#Unique Location Levels Cataloged	1

#Office	Name	Attribute	Alternate Names

LRL	CaesarCreek.Elev.Inst.0.Bottom of Flood Control		OH00927.Elev.Inst.0.Bottom of Flood Control
#Segment	Interpolate=True
#Date-Time America/Louisville	Elev (ft NGVD29)
04-Nov-2022 06:00	846.8832

**Comment by rma-rripken** [comment]
I don't have access to your exact server.  
The format parameter causes the end-point to basically pass-thru to a pl/sql procedure.  If you don't provide the format parameter but provide accept: application/json you still get the pass-thru to pl/sql
If you don't provide the format parameter and provide accept: application/json;version=2 you should get the output of the newer levels end-point. 

If you are using the newer end-point be careful with the begin/end.  I often forget to set them and get back []

**Comment by David-LRL** [comment]
Do you have access to this radar instance https://cwms-data.usace.army.mil/cwms-data?  I have tried the same queries on it and have gotten the same error responses.  This used to work.  I have also tried it with format parameters and the result regardless of which format I choose is:

{
  "message": "Formatting error",
  "incidentIdentifier": "1312471902669186103",
  "details": {}
}


From: rma-rripken ***@***.***>
Sent: Thursday, November 10, 2022 12:54 PM
To: USACE/cwms-radar-api ***@***.***>
Cc: Wilson, David L CIV USARMY CELRL (USA) ***@***.***>; Author ***@***.***>
Subject: [URL Verdict: Neutral][Non-DoD Source] Re: [USACE/cwms-radar-api] Formating issue with location levels (Issue #271)


I don't have access to your exact server.
The format parameter causes the end-point to basically pass-thru to a pl/sql procedure. If you don't provide the format parameter but provide accept: application/json you still get the pass-thru to pl/sql
If you don't provide the format parameter and provide accept: application/json;version=2 you should get the output of the newer levels end-point.

If you are using the newer end-point be careful with the start/end. I often forget to set them and get back []

**Comment by rma-rripken** [comment]
![2022-11-10_10-49-19](https://user-images.githubusercontent.com/89810919/201181675-3c69c675-f41c-446a-a806-04d7e65a44f6.png)
My last image was a capture of the wrong tab.  Sorry.  This is better.

Try doing it this way in the swagger-ui
Provide a begin time 
Dont put anything in the format parameter
Select Application/json:version=2 from the drop down

**Comment by David-LRL** [comment]
Tried your suggestion on both our local Radar and the https://cwms-data.usace.army.mil/cwms-data and got the same error response I have attached a screen shot.  At this point it appears to possibly be an issue with the Corps Build or the Corps environment.  Please coordinate with someone within USACE that has been part of the development process.

From: rma-rripken ***@***.***>
Sent: Thursday, November 10, 2022 1:55 PM
To: USACE/cwms-radar-api ***@***.***>
Cc: Wilson, David L CIV USARMY CELRL (USA) ***@***.***>; Author ***@***.***>
Subject: [URL Verdict: Neutral][Non-DoD Source] Re: [USACE/cwms-radar-api] Formating issue with location levels (Issue #271)


[Image removed by sender. 2022-11-10_10-49-19]<blockedhttps://user-images.githubusercontent.com/89810919/201181675-3c69c675-f41c-446a-a806-04d7e65a44f6.png>
My last image was a capture of the wrong tab. Sorry. This is better.

Try doing it this way in the swagger-ui
Provide a begin time
Dont put anything in the format parameter
Select Application/json:version=2 from the drop down

**Comment by rma-rripken** [comment]
@David-LRL if you attached a screenshot it must have been removed.  
The screenshot I posted earlier was from cwms-data.usace.army.mil/cwms-data/
Can you try the request from the command line?  
curl -X 'GET' \
  'https://cwms-data.usace.army.mil/cwms-data/levels?office=LRH&unit=EN&begin=2016-01-06T00%3A00%3A00Z' \
  -H 'accept: application/json;version=2'

When I issue the above command I get back:
{"page":"MHx8bnVsbHx8MTAw","levels":[{"location-level-id":"BeachCity.Stor.Inst.0.Top of Normal","office-id":"LRH","specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":2096919.1238307839,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Bottom of Normal","office-id":"LRH","specified-level-id":"Bottom of Normal","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":16035.263888117759,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Inactive","office-id":"LRH","specified-level-id":"Top of Inactive","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":16035.263888117759,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":3,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":3,"offset-minutes":0},{"value":1.1700808710975772E7,"offset-months":3,"offset-minutes":21600}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":3,"offset-minutes":0},{"value":1.1700808710975772E7,"offset-months":3,"offset-minutes":21600},{"value":5227496.02752639,"offset-months":0,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":3,"offset-minutes":0},{"value":1.1700808710975772E7,"offset-months":3,"offset-minutes":21600},{"value":5227496.02752639,"offset-months":0,"offset-minutes":0},{"value":5227496.02752639,"offset-months":11,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":3,"offset-minutes":0},{"value":1.1700808710975772E7,"offset-months":3,"offset-minutes":21600},{"value":5227496.02752639,"offset-months":0,"offset-minutes":0},{"value":5227496.02752639,"offset-months":11,"offset-minutes":0},{"value":1.1700808710975772E7,"offset-months":10,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":11,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":11,"offset-minutes":0},{"value":1.1824156894730527E7,"offset-months":10,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":11,"offset-minutes":0},{"value":1.1824156894730527E7,"offset-months":10,"offset-minutes":0},{"value":1.1824156894730527E7,"offset-months":3,"offset-minutes":1440}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":11,"offset-minutes":0},{"value":1.1824156894730527E7,"offset-months":10,"offset-minutes":0},{"value":1.1824156894730527E7,"offset-months":3,"offset-minutes":1440},{"value":5227496.02752639,"offset-months":0,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"BeechFk.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":5227496.02752639,"offset-months":11,"offset-minutes":0},{"value":1.1824156894730527E7,"offset-months":10,"offset-minutes":0},{"value":1.1824156894730527E7,"offset-months":3,"offset-minutes":1440},{"value":5227496.02752639,"offset-months":0,"offset-minutes":0},{"value":5227496.02752639,"offset-months":3,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2000-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"Dover.Stor.Inst.0.Bottom of Normal","office-id":"LRH","specified-level-id":"Bottom of Normal","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":0.0,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"Dover.Stor.Inst.0.Top of Inactive","office-id":"LRH","specified-level-id":"Top of Inactive","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":0.0,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"Dover.Stor.Inst.0.Top of Normal","office-id":"LRH","specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":0.0,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Bottom of Normal","office-id":"LRH","specified-level-id":"Bottom of Normal","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":12334.818375475199,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Inactive","office-id":"LRH","specified-level-id":"Top of Inactive","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":12334.818375475199,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":1.4444072317681458E7,"offset-months":0,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":1.4444072317681458E7,"offset-months":0,"offset-minutes":0},{"value":2.120108582376677E7,"offset-months":3,"offset-minutes":21600}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":1.4444072317681458E7,"offset-months":0,"offset-minutes":0},{"value":2.120108582376677E7,"offset-months":3,"offset-minutes":21600},{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":1.4444072317681458E7,"offset-months":0,"offset-minutes":0},{"value":2.120108582376677E7,"offset-months":3,"offset-minutes":21600},{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":11,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Normal","office-id":"LRH","seasonal-values":[{"value":1.4444072317681458E7,"offset-months":0,"offset-minutes":0},{"value":2.120108582376677E7,"offset-months":3,"offset-minutes":21600},{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":11,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":3,"offset-minutes":0}],"specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":11,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":11,"offset-minutes":0},{"value":2.120108582376677E7,"offset-months":3,"offset-minutes":21600}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":11,"offset-minutes":0},{"value":2.120108582376677E7,"offset-months":3,"offset-minutes":21600},{"value":1.4444072317681458E7,"offset-months":3,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"EastLynn.Stor.Inst.0.Top of Recreation","office-id":"LRH","seasonal-values":[{"value":2.120108582376677E7,"offset-months":10,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":11,"offset-minutes":0},{"value":2.120108582376677E7,"offset-months":3,"offset-minutes":21600},{"value":1.4444072317681458E7,"offset-months":3,"offset-minutes":0},{"value":1.4444072317681458E7,"offset-months":0,"offset-minutes":0}],"specified-level-id":"Top of Recreation","parameter-type-id":"Inst","parameter-id":"Stor","level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","interval-origin":"2021-01-01T05:00:00Z","interval-months":12,"interpolate-string":"T","duration-id":"0"},{"location-level-id":"JWFlannagan.Elev.Inst.0.Bottom of Conservation","office-id":"LRH","specified-level-id":"Bottom of Conservation","parameter-type-id":"Inst","parameter-id":"Elev","constant-value":374.904,"level-units-id":"m","level-date":"2018-03-01T18:00:00Z","duration-id":"0"},{"location-level-id":"JWFlannagan.Stor.Inst.0.Top of Inactive","office-id":"LRH","specified-level-id":"Top of Inactive","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":336740.54165047297,"level-units-id":"m3","level-date":"2021-04-01T15:00:00Z","duration-id":"0"},{"location-level-id":"Leesville.Elev.Inst.0.Top of Record","office-id":"LRH","specified-level-id":"Top of Record","parameter-type-id":"Inst","parameter-id":"Elev","constant-value":296.10100800000004,"level-units-id":"m","level-date":"2019-06-25T09:00:00Z","duration-id":"0"},{"location-level-id":"Mohawk.Stor.Inst.0.Bottom of Normal","office-id":"LRH","specified-level-id":"Bottom of Normal","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":123348.183754752,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"Mohawk.Stor.Inst.0.Top of Inactive","office-id":"LRH","specified-level-id":"Top of Inactive","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":123348.183754752,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"Mohawk.Stor.Inst.0.Top of Normal","office-id":"LRH","specified-level-id":"Top of Normal","parameter-type-id":"Inst","parameter-id":"Stor","constant-value":123348.183754752,"level-units-id":"m3","level-date":"2021-03-18T13:00:00Z","duration-id":"0"},{"location-level-id":"Paintsville.Elev.Inst.0.Top of Record","office-id":"LRH","specified-level-id":"Top of Record","parameter-type-id":"Inst","parameter-id":"Elev","constant-value":221.19945600000003,"level-units-id":"m","level-date":"2021-03-04T13:00:00Z","duration-id":"0"}],"page-size":100}


@MikeNeilson Lets coordinate - Is there a Corps system interfering with these requests?

**Comment by MikeNeilson** [comment]
I think our last update just introduced some bugs. I know I worked on the Format Header/Accept/Content-type processing. Beyond what I think is also a bug with the connections I'll put a test in for the formatting so we keep that default behavior.

**Comment by FHanbali** [comment]
Hey Mike, any update on this issue please?  Is there a new build that LRL can use?  This is a hang up on one of their tasks.

**Comment by MikeNeilson** [comment]
I think we fixed that header processing issue, but they'll need to try again with the latest version to know. I'll try to give it a check later after I've sorted through things today.

**Comment by rma-rripken** [comment]
What is the issue?  
I provided screenshots of how to use the swagger api to get data.  I provided an example of how to get json data from the command-line.  
Just now I tried the swagger-ui for https://wm.lrl.ds.usace.army.mil:8243/lrl-data/ and I get back json data.  
It seems to me that the end-point is currently working and has always been working.

Does LRL want back JSON formatted data?
Has LRL been able to get back JSON formatted data from the Swagger UI?
Is LRL calling this from Java or Python and need help adding the accept parameter?

**Comment by rma-rripken** [comment]
Happy to jump on a webex and sort out issues 1 on 1 if that is easier.

**Comment by David-LRL** [comment]
If we could do a quick Webex that would be great.

From: rma-rripken ***@***.***>
Sent: Monday, January 9, 2023 3:55 PM
To: USACE/cwms-radar-api ***@***.***>
Cc: Wilson, David L CIV USARMY CELRL (USA) ***@***.***>; Mention ***@***.***>
Subject: [URL Verdict: Neutral][Non-DoD Source] Re: [USACE/cwms-radar-api] Formating issue with location levels (Issue #271)


Happy to jump on a webex and sort out issues 1 on 1 if that is easier.

**Comment by rma-rripken** [comment]
I talked with David today.  The v2 end-point is working.  David had previously been using the v1 end-point and it stopped working for him for some reason.  In my test during the call the v1 end-point seemed to be working but I picked somewhat random arguments so I'm not sure if a bug had been fixed along the way or if David was just doing things differently than I was.

The v1 end-point is pretty much a straight pass-thru to pl/sql. Something could have changed on the database side of things too.

David is going to give it another try and if v1 isn't working he's going to look into using the v2 end-point.
